core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcToEnumerableConverter.java

```diff
@@ -323,6 +323,14 @@ private static void generateGet(EnumerableRelImplementor implementor,
               java.sql.Array.class);
       source = Expressions.call(BuiltInMethod.JDBC_ARRAY_TO_LIST.method, x);
       break;
+    case GEOMETRY:
```

Comment:
Are there tests that cover this path?

Reply:
We have usages of testcontainers in Calcite. Is there something that  prevents the integration tests to be committed as part of this PR?

Reply:
My integration tests are located here: https://github.com/bchapuis/calcite-postgis-tests

I looked into how Calcite uses Testcontainers, but it seems the support is currently limited to a few Redis tests. The JDBC tests don’t appear to leverage Testcontainers yet, which is why I set up external integration tests. Did I overlook anything?